composer.json

```diff
@@ -24,9 +24,15 @@
     "provide": {
         "jackalope/jackalope-transport": "1.3.0"
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "git@github.com:gabriel-caruso/phpcr-api-tests.git"
```

Comment:
hm, not sure whats wrong, but travis-ci does not like this. can you try https://github.com/gabriel-caruso/phpcr-api-tests.git ?